packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx

```diff
@@ -285,15 +286,14 @@ const DigitalContactsCodeVerificationProvider: FC<ReactNode> = ({ children }) =>
                 {t(`${modalProps.labelRoot}.${modalProps.labelType}-new-code`, { ns: 'recapiti' })}
                 &nbsp;
               </Typography>
-              <Typography
-                variant="body2"
-                display="inline"
-                color="primary"
+              <ButtonNaked
                 onClick={() => handleCodeVerification(undefined, true)}
-                sx={{ cursor: 'pointer' }}
+                sx={buttonNakedInheritStyle}
```

Comment:
Maybe this style is useless. Add only verticalAlign: unset and check that mobile and desktop views are ok

Reply:
Done